cernopendata_client/cli.py

```diff
@@ -55,6 +59,11 @@ def cernopendata_client():
 def get_record(recid, doi, title, output_fields):
     """Get records content by its recid, doi or title filtered."""
     # TODO: Add decorator to require one of (recid, doi or title)
+    if recid is not None and recid < 0: # This works
```

Comment:
Works well indeed :+1: except for the recid == 0 situation where one gets:

```console
$ cernopendata-client get-record --recid 0
Please provide at least one of following arguments: (recid, doi, title)
```

One should get the usual message:

```console
$ $ cernopendata-client get-record --recid -1
Usage: cernopendata-client get-record [OPTIONS]

Error: Invalid value for '--recid': Recid should be a positive integer
```